<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TetgenDialog</class>
 <widget class="QDialog" name="TetgenDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Tetgen Options</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>501</width>
     <height>251</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>501</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="volumeConstraintLabel">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Volume Constraint</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="volumeConstraintSpinBox">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>0.300000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="qualityBoundLabel">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Quality Bound</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="qualityBoundSpinBox">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="maximum">
          <double>90.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>1.500000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QCheckBox" name="usePlcCheckbox">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="toolTip">
          <string>This enables Tetgen's PLC reading (recommended to leave enabled)</string>
         </property>
         <property name="text">
          <string>Use PLC</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="generateTetrasButton">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Generate Tetrahedra</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TetgenDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TetgenDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>generateTetrasButton</sender>
   <signal>released()</signal>
   <receiver>TetgenDialog</receiver>
   <slot>generateTetras()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>displayMessage(QString,uint)</signal>
  <signal>notifyDone(QString,uint)</signal>
  <slot>generateTetras()</slot>
 </slots>
</ui>
